dockerfiles/mlrun-api/requirements.txt

```diff
@@ -14,3 +14,4 @@ alembic~=1.9
 timelength~=1.1
 memray~=1.12; sys_platform != 'win32'
 aiosmtplib~=3.0
+kfp~=1.8
```

Comment:
not needed if you have https://github.com/mlrun/mlrun/pull/6819/files#diff-36ba920c9a8bb3edbf8726c09b75edf3e46b0eddfb58406617c62b0356407d87R74 right?